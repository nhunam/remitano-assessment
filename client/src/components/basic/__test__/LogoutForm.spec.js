import LogoutForm from "components/basic/Form/Logout";
import React from "react";
import { shallow } from "enzyme";

describe("Logout Form", () => {
  const onLogoutMock = jest.fn();
  const mockUser = {
    info: { email: "test" }
  };
  const wrapper = shallow(
    <LogoutForm onLogout={onLogoutMock} user={mockUser} />
  );

  it("should render properly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
