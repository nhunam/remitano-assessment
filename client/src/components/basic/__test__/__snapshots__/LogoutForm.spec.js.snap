// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout Form should render properly 1`] = `ShallowWrapper {}`;
